build/build-plugins.js

```diff
@@ -59,76 +59,67 @@ const defaultPluginConfig = {
 }
 
 const getConfigByPluginKey = pluginKey => {
-  if (
-    pluginKey === 'Data' ||
-    pluginKey === 'Manipulator' ||
-    pluginKey === 'EventHandler' ||
-    pluginKey === 'SelectorEngine' ||
-    pluginKey === 'Util' ||
-    pluginKey === 'Sanitizer' ||
-    pluginKey === 'Backdrop'
-  ) {
-    return {
-      external: []
+  switch (pluginKey) {
+    case 'Alert':
+    case 'Offcanvas':
+    case 'Tab':
+      return defaultPluginConfig
+
+    case 'Base':
```

Comment:
Which component is this? 😖

Reply:
`BaseComponent` :thinking:
https://github.com/twbs/bootstrap/blob/b613c8dbb6c30e62adaefc669130549fd9c2e28b/build/build-plugins.js#L32